### PR TITLE
Add new module - junos_ping

### PR DESCRIPTION
**Pull request by NilashishC**
##### SUMMARY
- Targets FR #50497 

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
junos_ping.py

### CODE REVIEW

**Review by trishnaguha — APPROVED** [review]

File: lib/ansible/modules/network/junos/junos_ping.py [hunk 1]
```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Ansible by Red Hat, inc
```
Comment (trishnaguha) [inline lib/ansible/modules/network/junos/junos_ping.py]: Use 2019 instead and use the latest license template https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/cli/cli_config.py#L1-L5

**Review by trishnaguha — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/network/junos/junos_ping.py (hunk 1 above)
Comment (NilashishC) [reply lib/ansible/modules/network/junos/junos_ping.py]: Fixed.

**Review by trishnaguha — APPROVED** [review]

File: lib/ansible/modules/network/junos/junos_ping.py [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+# (c) 2019, Ansible by Red Hat, inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
```
Comment (ganeshrn) [inline lib/ansible/modules/network/junos/junos_ping.py]: This should be `network`?

File: lib/ansible/modules/network/junos/junos_ping.py [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+
+DOCUMENTATION = """
+---
+module: junos_ping
+short_description: Tests reachability using ping from devices running Juniper JUNOS
+description:
+  - Tests reachability using ping from a VyOS device to a remote destination.
```
Comment (ganeshrn) [inline lib/ansible/modules/network/junos/junos_ping.py]: Replace vyos with junos

File: lib/ansible/modules/network/junos/junos_ping.py [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+    count = module.params["count"]
+    dest = module.params["dest"]
+    source = module.params["source"]
+    size = module.params["size"]
+    ttl = module.params["ttl"]
+    interval = module.params["interval"]
+    interface = module.params['interface']
+    warnings = list()
+
+    results = {}
```
Comment (ganeshrn) [inline lib/ansible/modules/network/junos/junos_ping.py]: this can be initialized to `result = {'changed': False}`

**Comment by NilashishC** [comment]
@ganeshrn made the changes.

**Review by ganeshrn — APPROVED** [review]

**Pull request merged**